conventions/documentation/delete.md

```diff
@@ -17,6 +17,6 @@ When deleting a resource, a user must confirm by typing the resource name. After
 
 **Example (Pod):**
 
-Some resources do not require a confirmation by typing, but still must be confirmed via a confirmation modal.
+Some resources do not require a confirmation by typing, but still must be confirmed via a [confirmation modal](http://openshift.github.io/openshift-origin-design/conventions/documentation/alerting.html#confirmation-modal).
 
 ![deleting project](../images/delete-modal-alt.png)
```

Comment:
nit: I know this is an older example, but there is header text in this one and not in the other...should we be consistent? Of course, this could be a follow-up thing and not part of this PR necessarily :)

Reply:
Just updated these delete modals with the current versions